eav/models.py

```diff
@@ -288,6 +317,12 @@ class Meta:
         verbose_name=_('Created'),
     )
 
+    def natural_key(self):
```

Comment:
Same with these. Is there value there for having these methods vs doing `(attr.name, attr.slug)`?

Reply:
[Django Natural keys](https://docs.djangoproject.com/en/4.2/topics/serialization/#natural-keys).